src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -411,4 +427,81 @@ void CryptoManagerImpl::InitCertExpTime() {
   strptime("1 Jan 1970 00:00:00", "%d %b %Y %H:%M:%S", &expiration_time_);
 }
 
+bool CryptoManagerImpl::UpdateModuleCertificateData(X509* certificate,
+                                                    EVP_PKEY* key) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  if (NULL != certificate) {
```

Comment:
@AKalinich-Luxoft `if(certificate)`

Reply:
@AByzhynar fixed in 65a8cb2fce55a81ed184b172d4c9490ea00e1f3a